### ISSUE TITLE
Unable to start

### ISSUE THREAD

**Issue by ghost** [nicehash/NiceHashMiner-Archived#230]
I keep getting this error when I click "Start". Benchmarking and all other buttons work fine.

![image](https://cloud.githubusercontent.com/assets/3239204/18603622/3e2ccc32-7c41-11e6-8f7d-8c84083292d6.png)

I've never had any problems with version 1.6 and older. Here are the details:

``` text
See the end of this message for details on invoking 
just-in-time (JIT) debugging instead of this dialog box.

************** Exception Text **************
System.ArgumentOutOfRangeException: 'count' must be non-negative.
Parameter name: count
   at System.String.CtorCharCount(Char c, Int32 count)
   at NiceHashMiner.Miners.MinersManager.GetEnabledDeviceProifitDictionary(Dictionary`2 speedDict, Dictionary`2 NiceHashData)
   at NiceHashMiner.Miners.MinersManager.SwichMostProfitableGroupUpMethod(Dictionary`2 NiceHashData)
   at NiceHashMiner.Form_Main.SMAMinerCheck_Tick(Object sender, EventArgs e)
   at System.Windows.Forms.Timer.OnTick(EventArgs e)
   at System.Windows.Forms.Timer.TimerNativeWindow.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)


************** Loaded Assemblies **************
mscorlib
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.1080.0 built by: NETFXREL3STAGE
    CodeBase: file:///C:/Windows/Microsoft.NET/Framework/v4.0.30319/mscorlib.dll
----------------------------------------
NiceHashMiner
    Assembly Version: 1.7.1.3
    Win32 Version: 1.7.1.3
    CodeBase: file:///C:/NiceHashMiner_v1.7.1.3/NiceHashMiner.exe
----------------------------------------
System.Windows.Forms
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.1038.0 built by: NETFXREL2
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Windows.Forms/v4.0_4.0.0.0__b77a5c561934e089/System.Windows.Forms.dll
----------------------------------------
Newtonsoft.Json
    Assembly Version: 7.0.0.0
    Win32 Version: 7.0.1.18622
    CodeBase: file:///C:/NiceHashMiner_v1.7.1.3/Newtonsoft.Json.DLL
----------------------------------------
System
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.1081.0 built by: NETFXREL3STAGE
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System/v4.0_4.0.0.0__b77a5c561934e089/System.dll
----------------------------------------
System.Drawing
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.1078.0 built by: NETFXREL3STAGE
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Drawing/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Drawing.dll
----------------------------------------
System.Management
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.1038.0 built by: NETFXREL2
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Management/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Management.dll
----------------------------------------
System.Xml
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.1064.2 built by: NETFXREL3STAGE
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Xml/v4.0_4.0.0.0__b77a5c561934e089/System.Xml.dll
----------------------------------------
System.Data
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.1082.0 built by: NETFXREL3STAGE
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_32/System.Data/v4.0_4.0.0.0__b77a5c561934e089/System.Data.dll
----------------------------------------
System.Core
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.1084.0 built by: NETFXREL3STAGE
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Core/v4.0_4.0.0.0__b77a5c561934e089/System.Core.dll
----------------------------------------
log4net
    Assembly Version: 1.2.15.0
    Win32 Version: 1.2.15.0
    CodeBase: file:///C:/NiceHashMiner_v1.7.1.3/log4net.DLL
----------------------------------------
System.Configuration
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.1038.0 built by: NETFXREL2
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Configuration/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Configuration.dll
----------------------------------------
Accessibility
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.1038.0 built by: NETFXREL2
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/Accessibility/v4.0_4.0.0.0__b03f5f7f11d50a3a/Accessibility.dll
----------------------------------------
MessageBoxManager
    Assembly Version: 1.0.0.0
    Win32 Version: 1.0.0.0
    CodeBase: file:///C:/NiceHashMiner_v1.7.1.3/MessageBoxManager.DLL
----------------------------------------

************** JIT Debugging **************
To enable just-in-time (JIT) debugging, the .config file for this
application or computer (machine.config) must have the
jitDebugging value set in the system.windows.forms section.
The application must also be compiled with debugging
enabled.

For example:

<configuration>
    <system.windows.forms jitDebugging="true" />
</configuration>

When JIT debugging is enabled, any unhandled exception
will be sent to the JIT debugger registered on the computer
rather than be handled by this dialog box.
```

**Comment by S74nk0** [comment]
Hi @huangyihe. What locale/language is set in your system?

**Comment by ghost** [comment]
The locale is English (United States). The log file doesn't seem to contain any direct clues, but it's pasted down below anyway:

``` text
[2016-09-20 21:55:48] [INFO] [NICEHASH] Starting up NiceHashMiner v1.7.1.3
[2016-09-20 21:55:48] [INFO] [NICEHASH] Selected language: English
[2016-09-20 21:55:48] [INFO] [NICEHASH] Total RAM: 16315MB
[2016-09-20 21:55:48] [INFO] [NICEHASH] Page File Size: 200MB
[2016-09-20 21:55:48] [INFO] [ComputeDeviceQueryManager] 
QueryVideoControllers: 
    Win32_VideoController detected:
        Name NVIDIA GeForce GTX 950
        Description NVIDIA GeForce GTX 950
        PNPDeviceID PCI\VEN_10DE&DEV_1402&SUBSYS_85541043&REV_A1\4&1286464&0&0008
        DriverVersion 21.21.13.7270
        Status OK

[2016-09-20 21:55:48] [INFO] [MinersManager] Creating MinerPathChecker miners
[2016-09-20 21:55:48] [INFO] [AMD_OpenCL-MINER_ID(0)-DEVICE_IDs(NOT_SET)] NEW MINER CREATED
[2016-09-20 21:55:48] [INFO] [NVIDIA2.1-MINER_ID(1)-DEVICE_IDs(NOT_SET)] NEW MINER CREATED
[2016-09-20 21:55:48] [INFO] [NVIDIA3.x-MINER_ID(2)-DEVICE_IDs(NOT_SET)] NEW MINER CREATED
[2016-09-20 21:55:48] [INFO] [NVIDIA5.x-MINER_ID(3)-DEVICE_IDs(NOT_SET)] NEW MINER CREATED
[2016-09-20 21:55:48] [INFO] [NVIDIA6.x-MINER_ID(4)-DEVICE_IDs(NOT_SET)] NEW MINER CREATED
[2016-09-20 21:55:48] [INFO] [CPU0-MINER_ID(5)-DEVICE_IDs(NOT_SET)] NEW MINER CREATED
[2016-09-20 21:55:49] [INFO] [ComputeDeviceQueryManager] 
CudaDevicesDetection:
    ADDED device:
        ID: 0
        NAME: ASUS GeForce GTX 950
        VENDOR: ASUS
        UUID: GPU-04eca32f-b61b-a915-6b14-b98a0ae5f234
        SM: 5.2
        MEMORY: 2147483648
        ETHEREUM: YES

[2016-09-20 21:55:49] [INFO] [ComputeDeviceQueryManager] 
AMDOpenCLDeviceDetection found devices success:
    Found devices for platform: Intel(R) OpenCL
    Found devices for platform: NVIDIA CUDA
        Device:
            Device ID 0
            Device NAME GeForce GTX 950
            Device TYPE GPU

[2016-09-20 21:55:49] [INFO] [ComputeDeviceQueryManager] Checking AMD device (driver): NVIDIA GeForce GTX 950 (21.21.13.7270)
[2016-09-20 21:55:49] [INFO] [ComputeDeviceQueryManager] Cuda NVIDIA/CUDA device count GOOD
[2016-09-20 21:55:49] [INFO] [ComputeDeviceQueryManager] AMD GPU device count GOOD
[2016-09-20 21:55:49] [INFO] [NICEHASH] Version get
[2016-09-20 21:55:50] [INFO] [NICEHASH] SMA get
[2016-09-20 21:55:50] [INFO] [COINBASE] Bitcoin rate get
[2016-09-20 21:55:50] [INFO] [COINBASE] Current Bitcoin rate: 608.96
[2016-09-20 21:55:50] [INFO] [NICEHASH] Balance get
[2016-09-20 21:55:50] [INFO] [NICEHASH] Setting environment variables
[2016-09-20 21:55:50] [INFO] [NICEHASH] Trying to enable/disable Windows error reporting
[2016-09-20 21:55:50] [INFO] [NICEHASH] Current DontShowUI value: 1
```

**Comment by S74nk0** [comment]
Hi @huangyihe. This will be resolved with the future release.

**Comment by S74nk0** [comment]
Hi @huangyihe. This should be fixed in [v1.7.1.4](https://github.com/nicehash/NiceHashMiner/releases/tag/1.7.1.4). Please check and close this issue is resolved.

**Comment by ghost** [comment]
Yes it is resolved in v1.7.1.4. Thanks.
